packages/eslint-plugin-wpcalypso/lib/rules/i18n-unlocalized-url.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * @file Disallow using unlocalized URL strings
+ * @author Automattic
+ * @copyright 2023 Automattic. All rights reserved.
+ * See LICENSE.md file in root directory for full license.
+ */
+
+//------------------------------------------------------------------------------
+// Constants
+//------------------------------------------------------------------------------
+
+const LOCALIZABLE_URLS = [
+	// apps.wordpress.com/**
+	/^(https?:)?\/\/apps\.wordpress\.com/i,
+
+	// automattic.com/cookies/
+	/^(https?:)?\/\/automattic\.com\/cookies\/?$/i,
+
+	// automattic.com/privacy/
+	/^(https?:)?\/\/automattic\.com\/privacy\/?$/i,
```

Comment:
Should we support hashes? For some pages, we might want to lead to a particular section, for example:

https://automattic.com/privacy/#information-we-collect